### DISCUSSION TITLE
Where to find 7z.so?

### DISCUSSION THREAD

**Discussion by gbook** [rikyoz/bit7z#202] [General]
This may sound like a silly question, but I can't seem to find 7z.so on my RHEL8 system. I've installed p7zip, but there's no shared library.

I got my program working on Windows, and now I'm porting it to Linux, but... no lib!

Where can I find the 7z.so lib?

**Comment by rikyoz** [comment]
Hi!
Sorry for the late reply.
On Linux, the p7zip's 7z.so is usually installed at `/usr/lib/p7zip/7z.so`.
Please, be aware that for supporting p7zip, by default bit7z needs to be built with `BIT7Z_USE_LEGACY_IUNKNOWN` enabled (i.e., `-DBIT7Z_USE_LEGACY_IUNKNOWN=ON`).
